src/screens/Signup/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useForm } from 'react-hook-form';
+
+import logo from 'assets/logo.png';
+import Input from 'components/Input';
+import { requiredValidation } from 'utils/formValidations';
+
+import styles from './styles.module.scss';
+
+interface SignUpValues {
+  firstName: string;
+  lastName: string;
+  email: string;
+  password: string;
+  passwordConfirmation: string;
+}
+
+function Signup() {
+  const {
+    register,
+    formState: { errors },
+    handleSubmit
+  } = useForm<SignUpValues>();
+
+  const { t } = useTranslation();
+
+  const onSubmit = (data: SignUpValues) => {
```

Comment:
when you type `useForm` as you did, onSubmit payload is inferred, so this is unnecessary